kafka/adminapi.go

```diff
@@ -2737,6 +2738,9 @@ func (a *AdminClient) Close() {
 		return
 	}
 
+	if a.adminTermChan != nil {
```

Comment:
Add a Logs() method onto the AdminClient, too, which returns a.handle.logs, to be used in case `"go.logs.channel.enable": true,` but `"go.logs.channel": nil,` or unset.

After this, if `handle.closeLogsChan` is true, then also close the logs channel here.